platforms/software/maven/src/main/java/org/gradle/api/publish/maven/MavenPomDeploymentRepository.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.publish.maven;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.provider.Property;
+
+/**
+ * The repository information of the Maven distributionManagement node.
+ *
+ * @see MavenPom
+ * @see MavenPomDistributionManagement
+ * @since 8.12
+ */
+@Incubating
+public interface MavenPomDeploymentRepository {
+    /**
+     * A unique identifier for a repository.
```

Comment:
The docs are mirrored (copied) from https://maven.apache.org/ref/3.9.9/maven-model/maven.html#repository (Apache 2.0). Do you accept it or should I rewrite the docs?